routes/modelIndex.js

```diff
@@ -79,6 +78,41 @@ var route = function (req, res, next) {
             plural: req.linz.model.linz.formtools.model.plural
         },
         modelQuery: JSON.stringify(req.linz.model.formData)
+    };
+
+    const renderRecordAction = data.model.list.recordActions.renderer || recordActionRenderers.defaultRenderer;
+
+    data.records.forEach((record, index) => {
+
+        renderRecordAction({
```

Comment:
@smebberson I think we need something better here, calling `app.render` per record will cause a performance hit?